test/jsonapi.js

```diff
@@ -311,7 +320,7 @@ describe('Creation of new resources', () => {
   it('should add reverse relationship when inserting new resource', () => {
     const updatedState = reducer(state, apiCreated(taskWithTransaction));
 
-    const { data: taskRelationship } = updatedState.transactions.data[0].relationships.task;
+    const taskRelationship = updatedState.transactions.data[0].relationships.task;
 
     expect(taskRelationship.type).toEqual(taskWithTransaction.type);
     expect(taskRelationship.id).toEqual(taskWithTransaction.id);
```

Comment:
This doesn't seem right. Relationships should have a `data` property which points to a resource identifier.